### PR TITLE
Improve SEO for site & docs

### PR DESCRIPTION
**Pull request by ivarpruijn**
Site:
- Fixed meta description tag; can't contain ampersands.
- Swizzled the Layout component to remove unnecessary elements, and change the main div to a <main> element.
- Changed the header to a `<header>` element instead of a `<div>`.
- Changed the subtitle to an `<h2>` element instead of a `<p>`.

Docs:
- Fixed meta description tag; can't contain ampersands.
- Swizzled the Layout component to remove unnecessary elements, and change the main div to a <main> element.
- Main docs page (the introduction) now has title 'Documentation", as it will be index under "Objectiv [...]" anyway, and a description that explains what Objectiv is, instead of "Objectiv documentation logo" as a title and the "Welcome ..." text as a description.
- Also reworked the content on the main docs page to be more inline with the latest repo README.

### CODE REVIEW

File: docs/docs/introduction.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 sidebar_position: 1
 slug: /
-title: Introduction
+title: ""
```
Comment (ivarpruijn) [inline docs/docs/introduction.md:4]: Use main documentation title instead, which is now changed to "Documentation", as it's indexed under the main Objectiv page anyway.

File: docs/docs/introduction.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+description: Objectiv is open-source infrastructure for product analytics. Collect rich, model-ready data and feed it straight into your data warehouse. Cut down delivery times of data projects with reusable &amp; prebuilt models.
```
Comment (ivarpruijn) [inline docs/docs/introduction.md:5]: Now describes what Objectiv is in the meta tag.

File: docs/docs/introduction.md [hunk 3]
```diff
@@ -11,22 +12,19 @@ import Mermaid from '@theme/Mermaid';
 
 ![Objectiv Logo](/img/logo-objectiv-large.svg "Objectiv Logo")
 
-## Welcome to the official Objectiv documentation
-Objectiv is a complete, self-hosted product analytics stack designed for
-data science without gruntwork.
+## Objectiv Documentation
```
Comment (ivarpruijn) [inline docs/docs/introduction.md:15]: Reworked this page's content below to be more inline with the latest README.

File: docs/docusaurus.config.js [hunk 4]
```diff
@@ -13,9 +13,9 @@ const lightCodeTheme = require('prism-react-renderer/themes/github');
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 const config = {
-  title: "Objectiv - Open-source infrastructure for product analytics",
+  title: "Documentation",
```
Comment (ivarpruijn) [inline docs/docusaurus.config.js:16]: Title is now just "Documentation", as Google indexes it below the main Objectiv site anyway.

File: docs/docusaurus.config.js [hunk 5]
```diff
@@ -13,9 +13,9 @@ const lightCodeTheme = require('prism-react-renderer/themes/github');
...
   titleDelimiter: '|',
-  tagline: 'Collect rich, model-ready data and feed it straight into your data warehouse. Cut down delivery times of data projects with reusable & prebuilt models.', //meta description, and og:description
+  tagline: 'Objectiv is open-source infrastructure for product analytics. Collect rich, model-ready data and feed it straight into your data warehouse. Cut down delivery times of data projects with reusable &amp; prebuilt models.', //meta description, and og:description
```
Comment (ivarpruijn) [inline docs/docusaurus.config.js:18]: Apparently <meta> tags don't like ampersands

File: docs/src/theme/Layout/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+  useKeyboardNavigation();
+
+  return (
+    <LayoutProviders>
+      <LayoutHead {...props} />
+
+      <Navbar />
+
+      <main
```
Comment (ivarpruijn) [inline docs/src/theme/Layout/index.tsx:33]: Swizzled this component to remove some unnecessary elements, and turn this `<div>` into a `<main>`.

File: docusaurus.config.js [hunk 7]
```diff
@@ -13,7 +13,7 @@ const slackJoinLink = '/join-slack';
 const config = {
   title: "Objectiv - Open-source infrastructure for product analytics",
   titleDelimiter: '|',
-  tagline: 'Collect rich, model-ready data and feed it straight into your data warehouse. Cut down delivery times of data projects with reusable & prebuilt models.', //meta description, and og:description
+  tagline: 'Collect rich, model-ready data and feed it straight into your data warehouse. Cut down delivery times of data projects with reusable &amp; prebuilt models.', //meta description, and og:description
```
Comment (ivarpruijn) [inline docusaurus.config.js:16]: Meta tags don't like ampersands

File: src/theme/Layout/index.tsx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+  useKeyboardNavigation();
+
+  return (
+    <LayoutProviders>
+      <LayoutHead {...props} />
+
+      <Navbar />
+
+      <main
```
Comment (ivarpruijn) [inline src/theme/Layout/index.tsx:33]: Swizzled this component to remove some unnecessary elements, and turn this `<div>` into a `<main>`.

File: src/pages/index.tsx [hunk 9]
```diff
@@ -20,403 +20,400 @@ export default function Home() {
   const starUsNotificationAnchorRef = useRef(null);
 
   return (
-    <div>
```
Comment (ivarpruijn) [inline src/pages/index.tsx:23]: Removed this obsolete wrapping `<div>`, and now git thinks all the content below has changed, which is not the case.

File: src/pages/index.tsx [hunk 10]
```diff
@@ -20,403 +20,400 @@ export default function Home() {
...
-              for product analytics
-            </h1>
-            <p className={clsx(styles.heroSubTitle)}>
+    <Layout
+      title=''
+      description={tagline}>
+
+      <StarUsNotification innerRef={starUsNotificationAnchorRef} />
+      
+      <TrackedHeader
```
Comment (ivarpruijn) [inline src/pages/index.tsx:29]: Only other change: turned this into a `<header>`

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

**Comment by ivarpruijn** [comment]
Checked on testing, and looks good. Merging.

**Pull request merged**